azure/azure-pr-teardown-pipeline.yml

```diff
@@ -0,0 +1,55 @@
+name: "$(SourceBranchName)+$(BuildID)"
+
+trigger: none
+pr: none
+
+resources:
+  repositories:
+    - repository: common
+      type: github
+      name: NHSDigital/api-management-utils
+      ref: refs/heads/edge
+      endpoint: NHSDigital
+
+variables:
+  - template: project.yml
+
+jobs:
+  - job: teardown
+    displayName: PR Teardown
+    timeoutInMinutes: 30
+    pool:
+      name: 'AWS-ECS'
+      vmImage: 'ubuntu-latest'
+    steps:
+      - checkout: self
+
+      - bash: |
+          echo $(action_pr_number)
+          WORKSPACE="pr-"$(ACTION_PR_NUMBER)
+          echo $WORKSPACE
+          echo "##vso[task.setvariable variable=WORKSPACE]$WORKSPACE"
+          echo "##vso[task.setvariable variable=ACTION_PR_NUMBER]$(ACTION_PR_NUMBER)"
+        displayName: Set workspace tag
+
+      - template: "azure/components/aws-assume-role.yml@common"
+        parameters:
+            role: "auto-ops"
+            profile: "apim-dev"
+            aws_account: "dev"
+
+      - bash: |
+          echo "##vso[task.setvariable variable=AWS_DEFAULT_REGION]eu-west-2"
+        displayName: Set AWS default region
+
+      - bash: |
+          export AWS_PROFILE=apim-dev
+          account_id="$(aws sts get-caller-identity --query Account --output text)"
+
+          cd terraform
+          make init
+          # tf_vars has the client id, secret, and keycloak env required by default so we pass null values here to
```

Comment:
This comment is not true. Please remove all the unnecessary variables from terraform. If we need them in future we can always add them.